lib/perl/Genome/Config/AnalysisProject/Command/AddConfigFile.pm

```diff
@@ -70,4 +79,28 @@ sub _mark_instrument_data_bridges {
     return 1;
 }
 
+sub _apply_tags {
+    my ($self, $profile_item) = @_;
+    return unless $self->tag;
+    for my $tag_name ($self->tag){
+        my $tag = Genome::Config::Tag->create(name => $tag_name);
```

Comment:
`Genome::Config::Tag` is a table that holds a small, canonical set of tags.  The type of the `tag` parameter should be `Genome::Config::Tag` to save looking up or creating them here.